File: pset1/greedy.c
#include <stdio.h>
#include <cs50.h>

float change;
int cents, coins;

int main(void)
{
    do
    {
    printf("O hai! How much change is owed?\n");
    change = get_float();
    }
while (change < 0);

    cents = (change * 100);

while (cents >= 25)
    {
    coins ++;
    cents -= 25;
    }

while (cents >= 10)
    {
    coins ++;
    cents -= 10;
    }

while (cents >= 5)
    {
    coins ++;
    cents -= 5;
    }

    coins = cents + coins;
   
    printf("%i\n", coins);
}

File: pset6/greedy.py
while True:
    change = float(input("O hai! How much change is owed?\n"))
    if change >= 0:
        break
    
cents = change * 100
coins = 0
    
coins += cents // 25
cents = cents % 25

coins += cents // 10
cents = cents % 10

coins += cents // 5
cents = cents % 5

print(int(coins + cents))